.circleci/config.yml

```diff
@@ -4,7 +4,7 @@ default_steps: &default_steps
   steps:
     - checkout
     - run: yarn
-    - run: ./run-snapshots.sh
+    - run: npx percy exec -- mvn test
```

Comment:
In the other SDKs, we've removed visual tests since SDKs don't have visuals.

On that same vein, the PR mentions moving to GH actions, yet this Circle config still exists. Intentional?

Reply:
> Intentional?

Ha, no good catch. Ran out the door when I changed it from draft